### DISCUSSION TITLE
PTP - Print to pdf arranger

### DISCUSSION THREAD

**Discussion by mrserii** [pdfarranger/pdfarranger#943] [Ideas]
Im own small bussines and using the app a lot.

the major use for me is to a merge and arrange many docs to one (in my case - lawsuite).

when i attach a lot of docs from many source's like web, pdf, word, jpg, png i must mannually collect the doc's, than import to the app.

a virtual printer (print doc as pdf directly to arranger) is the most useful function i miss here.

something like : print to pdf arranger -> imports the printed doc to opened app ...

the arranger will act like docs clipboard ...

**Comment by jeromerobert** [comment]
I'm sorry but we won't implement that. Virtual printers are very system dependent and not implementable in python or with Poppler / GTK / PikePDF.

You can already copy / past file names and images. You can also drog & drop files from explorers.
